src/ray/raylet/worker_pool.cc

```diff
@@ -1,7 +1,6 @@
 #include "ray/raylet/worker_pool.h"
 
 #include <sys/wait.h>
-
```

Comment:
I think we want this newline according to https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes.

Reply:
@robertnishihara  OK